.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 .balena
 **/.env
+.idea/
```

Comment:
@matiasAS in the meantime, while we figure out how to handle out of tree commits in the tests, could you please take this line out.

Reply:
@ab77 is it ok now?

Reply:
Almost, just drop the new line from the end of that file.

Reply:
I'm not sure, but I think I deleted it, can you check?